client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/buffer/BufferPacker.java

```diff
@@ -114,7 +115,7 @@ public void close() {
   public static Queue<Buffer> unpack(ByteBuf byteBuf) {
     Queue<Buffer> buffers = new ArrayDeque<>();
     try {
-      Utils.checkState(byteBuf instanceof Buffer, "Illegal buffer type.");
+      checkState(byteBuf instanceof Buffer, "Illegal buffer type.");
```

Comment:
bytebuf requires the instance of  Buffer which is org.apache.flink.runtime.io.network.buffer.
 Unpack trace called is  getDataListener -> dataListener.accept(readData.getBuf()) -> new ReadData(streamId, backlog, offset, Unpooled.wrappedBuffer(tmpBuf). 
from the trace, it seems that it not the instance of  org.apache.flink.runtime.io.network.buffer